src/components/image-select.tsx

```diff
@@ -72,10 +72,8 @@ class ImageSelect extends React.Component<GridModalProps, GridModalState> {
   };
 
   getGridQueryString() {
-    const { cropWidth, cropHeight, label } = config.crop[
-      this.props.device || "mobile"
-    ];
-    return `?cropType=${label}&customRatio=${label},${cropWidth},${cropHeight}`;
+    const { cropRatio, label } = config.crop[this.props.device || "mobile"];
+    return `?cropType=${label}&customRatio=${label},10,${cropRatio * 10}`;
```

Comment:
I'm not sure the query string is really needed, I tried removing it from `getIframeUrl` and it didn't seem to make any difference.

Reply:
The query string tells the grid to limit the crop options to one cover card sized - see https://github.com/guardian/editions-card-builder/pull/86

Reply:
Discussed with @philmcmahon, this is still needed. It had persisted due to session storage.